CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [Unreleased]
+
+### Changed
+
+- Dropped the JSON-formatted `context` command-line parameter from Firecracker
+  in favor of individual classic command-line parameters.
+- Jailer no longer opens `/dev/kvm` and the API socket for Firecracker. These
```

Comment:
This doesn't look customer facing, so I don't think it belongs in the changelog. The information should be used to update `docs/jailer.md`, which is now out of date because it talks about the jailer opening devices and binding sockets :-s

Reply:
Makes sense, will do.

Reply:
done.